Mathlib/MeasureTheory/Measure/OuterMeasure.lean

```diff
@@ -290,76 +292,89 @@ variable [SMul R ℝ≥0∞] [IsScalarTower R ℝ≥0∞ ℝ≥0∞]
 
 variable [SMul R' ℝ≥0∞] [IsScalarTower R' ℝ≥0∞ ℝ≥0∞]
 
-instance : SMul R (OuterMeasure α) :=
+instance instSMul : SMul R (OuterMeasure α) :=
   ⟨fun c m =>
     { measureOf := fun s => c • m s
-      Empty := by rw [← smul_one_mul c (_ : ℝ≥0∞), empty', MulZeroClass.mul_zero]
-      mono := fun s t h => by
-        rw [← smul_one_mul c (m s), ← smul_one_mul c (m t)]
+      empty := by simp; rw [← smul_one_mul c]; simp
+      mono := fun {s t} h => by
+        simp
```

Comment:
Please remove the nonterminal simp

Reply:
For this one if I remove line 300, I get "tactic 'rewrite' failed, did not find instance of the pattern in the target expression
  c • ?y"